src/lib.rs

```diff
@@ -32,14 +32,21 @@
 #![forbid(unsafe_code)]
 #![warn(missing_docs, missing_debug_implementations, rust_2018_idioms)]
 
+#[cfg(not(all(loom, feature = "loom")))]
+use std::sync;
+
+#[cfg(all(loom, feature = "loom"))]
+use loom::sync;
+
 use std::cell::Cell;
 use std::fmt;
 use std::marker::PhantomData;
-use std::sync::atomic::AtomicUsize;
-use std::sync::atomic::Ordering::SeqCst;
-use std::sync::{Arc, Condvar, Mutex};
 use std::time::{Duration, Instant};
```

Comment:
Are these used when `#[cfg(loom)]` is active?

Reply:
Yes. I import either the `std::sync` or `loom::sync` module based on the configuration, and then I import synchronization primitives straight from the resulting `sync` module.

Reply:
nah, I'm talking about std::time::* stuff...

Reply:
`Duration` is used but `Instant` is not. This should now be fixed.